data-pipeline/src/trace_exporter.rs

```diff
@@ -157,10 +157,10 @@ impl TraceExporter {
 pub struct TraceExporterBuilder {
     host: Option<String>,
     port: Option<u16>,
-    tracer_version: Option<String>,
```

Comment:
The unwrapping of the tag values in the build function and the fact that the corresponding `TracerTag` values are not optional implies that they aren't actually optional. I switched them to `String` since that better signals intent and also leads to a very minor perf improvement in the builder (not a primary concern). 

If there are future plans that rely on these values truly being options I can change them back.